terraform-modules/http-load-balancer/variables.tf

```diff
@@ -25,24 +32,16 @@ variable "load_balancer_ssl_policy_enable" {
   default = "1"
 }
 
-variable "load_balancer_ssl_policy_create" {
-  default = "1"
-}
-
-variable "load_balancer_ssl_policy" {
-  default = "tls12-ssl-policy"
-}
-
 variable "load_balancer_instance_groups" {
   description = "Comma separated list of google self_links"
   default = ""
 }
 
 # Health check vars
 
-variable "load_balancer_health_check_url" {
+variable "load_balancer_health_check_path" {
```

Comment:
Note that this change will require calling code to change arguments (if it overrides this, which I think most things do not). I think it's worth it.